packages/edit-navigation/src/index.js

```diff
@@ -57,6 +57,7 @@ export function initialize( id, settings ) {
 		__experimentalRegisterExperimentalCoreBlocks( settings );
 	}
 	settings.__experimentalFetchLinkSuggestions = fetchLinkSuggestions;
+	settings.__experimentalNavigationManagementEditor = true;
```

Comment:
I think the notion of `screen` is widely used in WP? Like `ManagementEditor` sounds cool but kinda unexpected.

Reply:
Updated